articles/Remote Debugging/how-to-remote-debugging-with-vorlonJS.md

```diff
@@ -0,0 +1,33 @@
+<!--
+   tags:
+       Debugging:
+           - "Remote debugging with VorlonJS"
+-->
+# Remote debugging with VorlonJS
+
+> This article explains how to remotely debugging and testing your JavaScript using `vorlon.js`. While working with a Mac and an iPad might be straight forward, achievieving the same thing with a PC is more complicate. Here is one possible approach.
+
+## Server (your local machine)
+
+Install VorlonJS server from npm
+```console
+$ npm i -g vorlon
+```
```

Comment:
I also tried doing a local install of the package and it worked. But I don't have a problem with the global install here, as the package could be useful across many projects.